benchmark/README.md

```diff
@@ -2,7 +2,28 @@ Benchmarking
 ===
 
 # Generate Data
-> *TODO* instructions to be provided
+1. [Download](http://www.tpc.org/tpc_documents_current_versions/current_specifications.asp) the TPC-H benchmark tool.
+Follow the instructions for registration and download the tool and copy to local disk with free space larger than 300GB.
+
+2. Build the dbgen tool.
+    - Decompress the zip file, then navigate to `dbgen` folder.
+    - The TPC-H README contains instructions on how to build the tool in Linux.
+    - For Windows users, you can generate the dbgen.exe using Visual Studio: 
+        - i. In the `dbgen` folder, you will see `tpch.sln`, open it using Visual Studio (VS2015 or VS2017 should work).

```

Comment:
Tested vs2019, it worked. Discussed with @suhsteve offline that no need to specify the version since all the version worked. Also add that only need to build dbgen, no need to build qgen.